website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -0,0 +1,357 @@
+</Hds::Layout::Grid>
+```
+
+### Tag
+
+To specify the HTML tag used to render the grid container and/or item(s), use the `@tag` argument.
+
+```handlebars{data-execute=false}
+<Hds::Layout::Grid @tag="ul" as |LG|>
+  <li>{{! some content here }}</li>
+  <LG.Item @tag="li">
+    {{! some other content here }}
+  </LG.Item>
+  <li>{{! more content here }}</li>
+</Hds::Layout::Grid>
+```
+
+!!! Insight
+
+While, by default, the component renders a `<div>`, we invite consumers to consider which semantic HTML tag is the correct one for the context in which the text is used to meet WCAG Success Criterion [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) as the visual experience should match what is presented to the user with assistive technology.
+
+!!!
+
+### Spacing
+
+To control the spacing between grid items, use the `@gap` argument.
+
+Pass a single value to set equal spacing between columns & rows.
+
+```handlebars
+<Hds::Layout::Grid @columnMinWidth="50%" @gap="16">
+  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4c5f3" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e5ffd2" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#d2f4ff" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#fff8d2" />
+</Hds::Layout::Grid>
+```
+
+To differentiate the vertical and horizontal spacing between items when they wrap on multiple rows, provide an array of two values to the `@gap` argument.
+
+```handlebars
+<Hds::Layout::Grid @columnMinWidth="50%" @gap={{array "16" "48"}}>
+  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4c5f3" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e5ffd2" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#d2f4ff" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#fff8d2" />
+</Hds::Layout::Grid>
+```
+
+The first value in the array refers to the vertical gap between “rows” of items (`row-gap` in CSS), the second one to the horizontal spacing between “columns” of items (`column-gap` in CSS).
+
+The `@gap` argument accepts only **pre-defined values**, it can’t be used to provide custom spacing values. Refer to the [Component API](#component-api) section for details on which values are accepted.
+
+If you need to provide custom spacing values, see below how you can use a special escape hatch for this.
+
+!!! Warning
+
+**Important**
+
+The **pre-defined value(s)** passed to the `@gap` argument **must be string(s)**, not numbers!
+
+!!!
+
+#### Non-standard gap values
+
+If you absolutely have to use non-standard spacing value(s) for the grid `gap`, you can use the internal `--hds-layout-grid-row-gap` and `--hds-layout-grid-column-gap` CSS variables and pass custom values to them (e.g., via a local CSS variable or an inline style).
+
+```handlebars{data-execute=false}
+<Hds::Layout::Grid class="doc-grid-demo-custom-grid-column-gap">
+  {{!
```

Comment:
Suggestion: in the flex docs, theres a snippet of CSS in the content here so consumers can see what the class `doc-grid-demo-custom-grid-column-gap` does